sdk/digitaltwins/digitaltwins/README.md

```diff
@@ -29,28 +31,36 @@ npm install @azure/ms-rest-nodeauth
 import * as coreHttp from "@azure/core-http";
 import * as coreArm from "@azure/core-arm";
 import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { AzureDigitalTwinsAPI, AzureDigitalTwinsAPIModels, AzureDigitalTwinsAPIMappers } from "@azure/digitaltwins";
+import {
+  AzureDigitalTwinsAPI,
+  AzureDigitalTwinsAPIModels,
+  AzureDigitalTwinsAPIMappers
+} from "@azure/digitaltwins";
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new AzureDigitalTwinsAPI(creds, subscriptionId);
-  const dependenciesFor = ["testdependenciesFor"];
-  const includeModelDefinition = true;
-  const maxItemCount = 1;
-  client.digitalTwinModels.list(dependenciesFor, includeModelDefinition, maxItemCount).then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
-}).catch((err) => {
-  console.error(err);
-});
+msRestNodeAuth
+  .interactiveLogin()
+  .then((creds) => {
```

Comment:
would be nice to show this with async functions and `await`, but I'm fine with doing that later.

Reply:
I made a note.